mediathread/djangosherd/models.py

```diff
@@ -20,14 +20,14 @@
 from structuredcollaboration.models import Collaboration
 
 
-NULL_FIELDS = dict((i, None) for i in
-                   'range1 range2 title'.split())
+NULL_FIELDS = {'title': None}
```

Comment:
NULL_FIELDS is referenced in `djangosherd`'s `create_annotation`. (https://github.com/ccnmtl/mediathread/blob/master/mediathread/djangosherd/views.py#L19)

I *think* the `create_annotation` code is making use of NULL_FIELDS to ensure the user is specifying a portion of an asset rather than the "whole" asset. But likely worth tracing through to determine exact behavior.

Reply:
Yeah, I noticed that. I removed range1 and range2 here in order to make some tests pass and just clean things up, because this NULL_FIELDS is used in this file as well.

I've added a commit that uses the original version of NULL_FIELDS in djangosherd/models.py to account for this cliipping thing.. let me know if that doesn't make sense or you think of a better way to go about this.

Reply:
I'm also just going to remove NULL_FIELDS from this models.py file, as it's really not necessary.